test/itemlist_formaction.cpp

```diff
@@ -383,50 +376,88 @@ TEST_CASE("process_op(OP_SHOWURLS)", "[itemlist_formaction]") {
 	itemlist_formaction itemlist(&v, itemlist_str);
 
 
-	SECTION("OP_SHOWURLS with external-url-viewer"){
+	SECTION("with external-url-viewer"){
 		feed->add_item(item);
 		itemlist.set_feed(feed);
 		cfg.set_configvalue("external-url-viewer", "tee > " + urlFile.getPath());
 	
 		REQUIRE_NOTHROW(itemlist.process_op(OP_SHOWURLS));
 
 		std::ifstream urlFileStream (urlFile.getPath());
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == pager_prefix_title + test_title);
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == pager_prefix_author + test_author);
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == pager_prefix_date + test_pubDate_str);
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == pager_prefix_link + test_url);
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == " ");
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == test_description);
 
-		REQUIRE( std::getline (urlFileStream,line) );
+		REQUIRE(std::getline (urlFileStream,line));
 		REQUIRE(line == "");
 	}
 
-	SECTION("OP_SHOWURLS default behaviour"){
+	SECTION("default behaviour"){
 		feed->add_item(item);
 		itemlist.set_feed(feed);
 		REQUIRE_NOTHROW(itemlist.process_op(OP_SHOWURLS));
 	}
 
-	SECTION("OP_SHOWURLS empty feed"){
+	SECTION("no feed in formaction"){
 		REQUIRE_NOTHROW(itemlist.process_op(OP_SHOWURLS));
 	}
 	
 }
 
-TEST_CASE("process_op(OP_BOOKMARK)", "[itemlist_formaction]") {
-	//NOTIMPL
+TEST_CASE("OP_BOOKMARK works properly", "[itemlist_formaction]") {
```

Comment:
This, too, should be specific: "OP_BOOKMARK pipes item's URL to bookmark-cmd" or something.